### PR TITLE
Added styles for Card component

### PR DESCRIPTION
**Pull request by alexpricedev**
image

### CODE REVIEW

File: packages/open-ui/src/components/Card/Card.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import { Title } from 'components/Typography';
+import { verticalRhythm } from 'constants/styles';
+import styles from './Card.module.scss';
+
+interface CardProps {
+  title?: string;
+}
+
+export const Card: React.FC<CardProps & MarginStyles> = props => {
+  const { children, title, ...styleProps } = props;
```
Comment (gurjit03) [inline packages/open-ui/src/components/Card/Card.tsx]: @remotealex should consider renaming it to marginStyles ? Cause later on we might can consider adding the props.

File: packages/open-ui/src/components/Card/Card.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+  // Build the class names
+  const cls = classNames(styles.card);
+
+  // Build up the margin styles using our vertical rhythm
+  const _styleProps = {} as any;
+  Object.entries(styleProps).map(([key, value]: [string, number]) => {
```
Comment (gurjit03) [inline packages/open-ui/src/components/Card/Card.tsx]: Also was thinking to abstract this function to new folder utils or helpers. Whatcha say?

**Review by gurjit03 — CHANGES_REQUESTED** [review]

File: packages/open-ui/src/components/Card/Card.tsx (hunk 1 above)
Comment (alexpricedev) [reply packages/open-ui/src/components/Card/Card.tsx]: Yeah, good shout

File: packages/open-ui/src/components/Card/Card.tsx (hunk 2 above)
Comment (alexpricedev) [reply packages/open-ui/src/components/Card/Card.tsx]: yeah totally!

**Comment by alexpricedev** [comment]
Fixed in https://github.com/remotealex/open-build/pull/5

**Pull request merged**